namer/consul/src/test/scala/io/buoyant/namer/consul/ConsulNamerTest.scala

```diff
@@ -715,4 +715,60 @@ class ConsulNamerTest extends FunSuite with Awaits {
       )
     }
   }
+
+  test("Namer returns weighted bound address metadata when service has configured weight tag") {
+    class TestApi extends CatalogApi(null, "/v1") {
+      override def serviceMap(
+                               datacenter: Option[String] = None,
```

Comment:
this is not typically how we align parameters.  these should be indented one level from the line with the def.  take a look at the surrounding code for examples.

Reply:
I like to delegate the formatting to my IDE. Do you have a settings file for IntelliJ ?